Core/Services/Auth/LoginService.cs

```diff
@@ -1,32 +1,18 @@
-﻿using Core.Common;
-using Core.Interfaces.Auth;
+﻿using Core.Interfaces.Auth;
 using Core.Requests;
 using Core.ServiceResponses;
-using Core.Services.Email;
 using Microsoft.AspNetCore.Identity;
 using Microsoft.Extensions.Configuration;
-using Microsoft.IdentityModel.Tokens;
 using Storage.Identity;
-using System;
-using System.Collections.Generic;
-using System.IdentityModel.Tokens.Jwt;
 using System.Net;
-using System.Security.Claims;
-using System.Text;
 using System.Threading.Tasks;
 
 namespace Core.Services.Auth
 {
-    public class LoginService : ILoginService
+    class LoginService : AuthServicesProvider, ILoginService
```

Comment:
Is the default internal access modifier correct for this class?

Reply:
Yes, because I use this class only in this folder